src/core/lib/security/credentials/tls/grpc_tls_credentials_options.cc

```diff
@@ -149,3 +149,15 @@ void grpc_tls_credentials_options_set_crl_provider(
   GPR_ASSERT(options != nullptr);
   options->set_crl_provider(provider);
 }
+
+void grpc_tls_credentials_options_set_min_tls_version(
```

Comment:
Those fields are already there in C-core layer so that python script was taken care of before IIUC. The methods I introduce are just for the C++ or other wrapping langs to set the versions, as the option itself is opaque in the header.